### PR TITLE
AWS lambda renderer

### PR DESCRIPTION
**Pull request by oblador**
For very large test suite, running them on a single machine might be a bit slow. This PR introduces the concept of remote rendering, for now limited to AWS Lambdas to allow for a massive parallelisation (I run it at ~500 concurrent renders). There's a few params to tweak the execution/concurrency depending on your project.

### CODE REVIEW

File: packages/renderer-aws-lambda/src/index.spec.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const mod = require('./index');
```
Comment (techeverri) [inline packages/renderer-aws-lambda/src/index.spec.js]: Just out of curiosity 😬 

What does `mod` mean? 🤔

File: packages/target-chrome-core/src/create-chrome-target.js [hunk 2]
```diff
@@ -176,7 +178,7 @@ function createChromeTarget(
       }
     };
 
-    client.captureScreenshot = withRetries(options.chromeRetries)(
+    client.captureScreenshot = withRetries(options.chromeRetries, 500)(
```
Comment (techeverri) [inline packages/target-chrome-core/src/create-chrome-target.js]: We could give `500`  a nice name and move it to the top of the module

What do you think? 🤔

File: packages/target-native-android-emulator/src/create-android-emulator-target.js [hunk 3]
```diff
@@ -1,9 +1,17 @@
...
+const tempy = require('tempy');
 const osnap = require('osnap/src/android');
 const { createWebsocketTarget } = require('@loki/target-native-core');
 
-const saveScreenshotToFile = filename => osnap.saveToFile({ filename });
+const captureScreenshot = async () => {
+  const filename = tempy.file({ extension: 'png' });
+  await osnap.saveToFile({ filename });
+  const screenshot = await fs.readFile(filename);
+  await fs.unlink(filename);
```
Comment (techeverri) [inline packages/target-native-android-emulator/src/create-android-emulator-target.js]: What if `fs.readFile` or  `fs.unlink` fail?

Should this be wrapped in a `try-catch` block to handle (e.g. log, report, etc) the error?

File: packages/renderer-aws-lambda/src/index.spec.js (hunk 1 above)
Comment (oblador) [reply packages/renderer-aws-lambda/src/index.spec.js]: `module`, but it's a reserved word, so you can't/shouldn't have a variable with that name.

File: packages/target-native-android-emulator/src/create-android-emulator-target.js (hunk 3 above)
Comment (oblador) [reply packages/target-native-android-emulator/src/create-android-emulator-target.js]: The try-catch happens on the runner level, if the screenshot fails the story will fail which is correct behavior IMHO. You can't really continue with the comparison after this.

File: packages/target-chrome-core/src/create-chrome-target.js (hunk 2 above)
Comment (oblador) [reply packages/target-chrome-core/src/create-chrome-target.js]: Fixed!

File: packages/target-native-android-emulator/src/create-android-emulator-target.js (hunk 3 above)
Comment (techeverri) [reply packages/target-native-android-emulator/src/create-android-emulator-target.js]: If there is already a top-level `try-catch` block, it makes sense :+1:

**Pull request merged**